packages/vx-grid/src/grids/GridColumns.tsx

```diff
@@ -31,6 +38,10 @@ export default function GridColumns<ScaleInput>({
   lineStyle,
   offset,
   tickValues,
+  gridColumnLineProps = (/** tickValue, index */) => ({
```

Comment:
I think if you update to the spread logic I suggest below, you don't need a default for this prop (could use optional chaining on invocation `gridLineProps?.()`)